source/_snippet.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="snippet">
+  <header class="snippet-header">
```

Comment:
I think we can remove the `header` here. If this is needed for styling, a `div` would be a solid replacement.

The reason is that `header` comes with an implicit role of `banner`, and every `header` element then gets surfaced in things like the rotor menu found in VoiceOver (think of it like a way to quickly navigate around a page's landmarks). It looks like this:

<img width="1440" alt="screen shot 2017-01-13 at 14 08 00" src="https://cloud.githubusercontent.com/assets/903327/21942156/de466254-d999-11e6-8d14-929ba9e917b7.png">